### PR TITLE
Add enterprise manager

### PR DESCRIPTION
**Pull request by mattwiller**
/cc @djordan

### CODE REVIEW

File: docs/enterprise.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+* [Get Enterprise Users](#get-enterprise-users)
+* [Invite User to Enterprise](#invite-user-to-enterprise)
+* [Create New User](#create-new-user)
+
+Get Enterprise Users
+--------------------
+
+Get a list of users in the current enterprise by calling the
+[`enterprise.getUser(options, callback)`](http://opensource.box.com/box-node-sdk/Enterprise.html#getUsers)
```
Comment (djordan) [inline docs/enterprise.md]: Typo: getUsers

File: docs/enterprise.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+Invite a user to an enterprise by calling the
+[`enterprise.inviteUser(enterpriseID, email, callback)`](http://opensource.box.com/box-node-sdk/Enterprise.html#inviteUser)
+method with the ID of the enterprise and the user's email address.
+
+```js
+client.enterprise.inviteUser('1345', 'jsmith@box.com', callback);
+```
+
+Create New User
```
Comment (djordan) [inline docs/enterprise.md]: "Add New User"?

File: lib/managers/enterprise.js [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+ * @param {?Object} options - Optional parameters, can be left null in most cases
+ * @param {EnterpriseRole} [options.role] - This user’s enterprise role
+ * @param {string} [options.language] - The user's language
+ * @param {bool} [options.is_sync_enabled] - Whether or not this user can use Box Sync
+ * @param {string} [options.job_title] - The user’s job title
+ * @param {string} [options.phone] - The user’s phone number
+ * @param {string} [options.address] - The user’s address
+ * @param {int} [options.space_amount] - The user’s total available storage space in bytes
+ * @param {Array} [options.tracking_codes] - An array of key/value pairs set by the user’s admin
+ * @param {EnterpriseUserStatus} [options.status] - The user's status
```
Comment (djordan) [inline lib/managers/enterprise.js]: The docs don't list "status" as a param (https://docs.box.com/reference#create-an-enterprise-user), though it does come back in the response. Can it be set via API?

Comment (mattwiller) [inline lib/managers/enterprise.js]: Yeah, I think the docs are messed up

**Pull request merged**